sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -401,6 +401,78 @@ case class DayOfYear(child: Expression) extends UnaryExpression with ImplicitCas
   }
 }
 
+@ExpressionDescription(
+  usage = "_FUNC_(seconds) - Creates timestamp from the number of seconds since UTC epoch.",
+  examples = """
+    Examples:
+      > SELECT _FUNC_(1230219000);
+       "2008-12-25 07:30:00"
+  """,
+  group = "datetime_funcs",
+  since = "3.1.0")
+case class SecondsToTimestamp(child: Expression)
+  extends NumberToTimestampBase {
+
+  override def upScaleFactor: SQLTimestamp = MICROS_PER_SECOND
```

Comment:
Is that factor really a timestamp? That seems weird, it's just a Long in the base class.

Reply:
Done

Reply:
Yes, We need just a param, so look around I found the `downScaleFactor `, so the `upScaleFactor` is used, Could you please give any suggest, could it be `upScaleParam`?

Reply:
No, I mean that the type should be Long, just like the base class:
```suggestion
  override def upScaleFactor: Long = MICROS_PER_SECOND
```

Reply:
Ok, I will correct it, good job.

Reply:
+1